### PR TITLE
Rewrite README.md

### PR DESCRIPTION
**Pull request by akaroml**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,54 @@
...
+## Other Recommendations
 
-![Debugger for Java](https://raw.githubusercontent.com/Microsoft/vscode-java-pack/master/vscode-java-debug.gif)
+You can do more with VS Code. Here are some more recommendations that could help.
 
-- [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) - Run and debug JUnit test cases.
+### Spring
 
-![Java Test Runner](https://raw.githubusercontent.com/Microsoft/vscode-java-pack/master/vscode-java-test.gif)
+Spring Tools 4 (ST4) is also available in Visual Studio Code. It understands Spring so you can navigate Spring code at the level of beans, routes, and etc. It can also show live information of the running Spring Boot applications. Check out [the ST4 website](https://spring.io/tools) to see a complete list of its features.
```
Comment (andxu) [inline README.md]: and etc => etc

File: README.md [hunk 2]
```diff
@@ -1,24 +1,54 @@
...
-![Maven Project Explorer](https://raw.githubusercontent.com/Microsoft/vscode-java-pack/master/vscode-maven.gif)
+### Containers and Microservices
+
+You can use [📦 Docker](https://marketplace.visualstudio.com/items?itemName=PeterJausovec.vscode-docker) extension to build docker images and work with image registries.
+
+[📦 Kubernetes](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) extension provides an explorer view to manage clusters and the nodes inside. It also provides advanced syntax support for editing Kubernetes manifest files.
+
+### Tomcat and Jetty
+
+Both [📦 Tomcat](https://marketplace.visualstudio.com/items?itemName=adashen.vscode-tomcat) and [📦 Jetty](https://marketplace.visualstudio.com/items?itemName=SummerSun.vscode-jetty) extension are available. They provide dedicated views to help work with your favorite web servers.
```
Comment (andxu) [inline README.md]: help work => help you to work

File: README.md [hunk 3]
```diff
@@ -1,24 +1,54 @@
...
+By installing Java Extension Pack, the following extensions are installed:
 
-![Language Support for Java(TM) by Red Hat](https://raw.githubusercontent.com/Microsoft/vscode-java-pack/master/vscode-java.gif)
+- [📦 Language Support for Java™ by Red Hat ](https://marketplace.visualstudio.com/items?itemName=redhat.java)
+    - Code Navigation
+    - Auto Completion
+    - Refactoring
+    - Code Snippets
+- [📦 Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug)
+    - Debugging
```
Comment (testforstephen) [inline README.md]: - Run & Debug Java program

Comment (akaroml) [reply README.md]: Here we need a noun to be consistent with others.

File: README.md (hunk 2 above)
Comment (akaroml) [reply README.md]: There's no error in this sentence.

File: README.md (hunk 1 above)
Comment (akaroml) [reply README.md]: changed

**Pull request merged**